content/psy_recommend.md

```diff
@@ -0,0 +1,7 @@
+title: Психологи рекомендут
+slug: psy_recommend
```

Comment:
This `slug` name is ugly and should contain a dash (`-`) instead of an underscore (`_`). Rename it to `psychologists-recommend`.